### ISSUE TITLE
JAWS does not announce aria-describedby or aria-required when contenteditable="true"  elements have content

### ISSUE THREAD

**Issue by greypants** [FreedomScientific/VFO-standards-support#643]
### Summary

JAWS does not announce `aria-describedby` or `aria-required` when `contenteditable="true"` when the container has content.

**Works (label, description, and required are read):**
```html
<label id="label">My Label</label>
<div 
  role="textbox" 
  contenteditable="true" 
  tabindex="0" 
  aria-labelledby="label" 
  aria-describedby="description" 
  aria-required="true">
</div>
<p id="description">My description</p>
```

**Fails when text content is present (only label is read):**
```html
<label id="label">My Label</label>
<div 
   role="textbox" 
  contenteditable="true" 
  tabindex="0" 
  aria-labelledby="label" 
  aria-describedby="description" 
  aria-required="true">
    Some content
</div>
<p id="description">My description</p>
```
 
1. Go to https://codepen.io/greypants/pen/LYQgqLm?editors=1100
2. Tab through the examples, and note what JAWS reads

### Expected result

All examples should read the label, description, and required status when the textbox is focused.

### Actual result

Examples 4 and 5 do not read the description or required status.

### Example

https://codepen.io/greypants/pen/LYQgqLm?editors=1100

### Additional Information

The behavior works properly on the other screen readers I'm testing:
- VoiceOver/Safari/Mac OS 12 on Mac
- NVDA/Edge/Windows 10

#### JAWS version and build number
2022.2202.38 LIM

 
#### Operating System and version
Windows 10 (via Parallels desktop VM on Mac OS 12.3.1)

#### Browser and version:
Microsoft Edge 100.0.1185.39
